### ISSUE TITLE
Remove doc/tags from git repo?

### ISSUE THREAD

**Issue by bronson** [timcharper/textile.vim#4]
Hi, any chance you could remove doc/tags from textile.vim's git repo?  It's often wrong, which makes it difficult to pull upstream changes.

For instance, I just cloned your plugin fresh and generated helptags.  Now textile.vim's working directory contains these changes:

```
--- a/doc/tags
+++ b/doc/tags
@@ -1,5 +1,6 @@
 textile-changelog      textile.txt     /*textile-changelog*
 textile-commands       textile.txt     /*textile-commands*
+textile-config textile.txt     /*textile-config*
 textile-credits        textile.txt     /*textile-credits*
 textile-requirements   textile.txt     /*textile-requirements*
 textile.txt    textile.txt     /*textile.txt*
```

And when I run git pull I get:

```
Cannot pull: You have unstaged changes.
Please commit or stash them.
```

It seems that most vim plugins don't include a doc/tags file, and even put them in .gitignore.  This works great for me.

Unfortunately, if a file both exists and is listed in .gitignore, git will not ignore it.  So, it's impossible to ignore textile.vim's doc/tags file.

Does this make sense?  Just trying to avoid having to git reset --hard HEAD in ~/.vim/bundle/textile.vim before I can pull.

Thanks!

**Comment by timcharper** [comment]
On Jun 27, 2011, at 13:36 , bronson wrote:

> Hi, any chance you could remove doc/tags from textile.vim's git repo?  It's often wrong, which makes it difficult to pull upstream changes.
> 
> For instance, I just cloned your plugin fresh and generated helptags.  Now textile.vim's working directory contains these changes:

It is done :)

**Comment by bronson** [comment]
Thanks!!  Closed by 5d25629d066d01f3c9e0
